packages/valory/skills/agent_performance_summary_abci/handlers.py

```diff
@@ -39,12 +49,334 @@
+            safe_address = self.synchronized_data.safe_contract_address.lower()
+            
+            # Prepare the GraphQL query
+            query_payload = {
+                "query": GET_TRADER_AGENT_DETAILS_QUERY,
+                "variables": {"id": safe_address}
+            }
+            
+            # Get the subgraph URL from params
+            subgraph_url = self.context.params.olas_agents_subgraph_url
+            
+            # Make synchronous request to subgraph
+            response = requests.post(
+                subgraph_url,
+                json=query_payload,
+                headers={"Content-Type": "application/json"},
+                timeout=30
+            )
+            
+            if response.status_code != 200:
+                self.context.logger.error(
+                    f"Failed to fetch agent details from subgraph: {response.status_code}"
+                )
+                self._send_internal_server_error_response(
+                    http_msg,
+                    http_dialogue,
+                    {"error": "Failed to fetch agent details from subgraph"}
+                )
+                return
+            
+            response_data = response.json()
+            
+            # Extract trader agent data
+            trader_agent = response_data.get("data", {}).get("traderAgent")
+            
+            if not trader_agent:
+                self.context.logger.warning(
+                    f"No trader agent found for address: {safe_address}"
+                )
+                self._send_not_found_response(
+                    http_msg,
+                    http_dialogue
+                )
+                return
+            
+            # Format the response according to API spec
+            formatted_response = {
+                "id": trader_agent.get("id", safe_address),
+                "created_at": self._format_timestamp(trader_agent.get("blockTimestamp")),
+                "last_active_at": self._format_timestamp(trader_agent.get("lastActive")),
+            }
+            
+            self.context.logger.info(f"Sending agent details: {formatted_response}")
+            self._send_ok_response(http_msg, http_dialogue, formatted_response)
+            
+        except Exception as e:
+            self.context.logger.error(f"Error fetching agent details: {str(e)}")
+            self._send_internal_server_error_response(
+                http_msg,
+                http_dialogue,
+                {"error": "Internal server error while fetching agent details"}
+            )
+
+    def _format_timestamp(self, timestamp: str) -> str:
+        """
+        Format a Unix timestamp to ISO 8601 format.
+        
+        :param timestamp: Unix timestamp as string
+        :return: ISO 8601 formatted timestamp
+        """
+        if not timestamp:
+            return ""
+        
+        try:
+            # Convert to int if it's a string
+            timestamp_int = int(timestamp)
```

Comment:
Please rename to `unix_timestamp` or `timestamp_sec` etc., type should not be in the name.